salviosage/EVaccinefrontpage
# adding requred stuuf for your ui

**What does this PR do?**

- adding all required UI views

**Description of Task to be completed?**

- all UI template can be used to develop whole front views of the application 

**How should this be manually tested?**

- after cloning this repo you can open whichever HTML file in there and see what is included 

**What are the relevant pivotal tracker stories?**

- none
